src/views/Datasets.vue

```diff
@@ -166,103 +158,76 @@ const fakeFetch = (data, delay = 0) => {
 	})
 }
 
-
-// apiProvider fills the table with datasets from an (real or fake) API response.
-// Function is passed (ctx, callback).
-function apiProvider() {
-	let promise = (process.env.VUE_APP_ENVIRONMENT === 'development' && !process.env.VUE_APP_QVAIN_API_URL) ? fakeClient() : apiClient.get("/datasets/")
-	this.error = null
-	return promise.then((response) => {
-		console.log("api count:", (response.data || []).length)
-		return (response.data || [])
-	})
-		.catch((error) => {
-			this.error = getApiError(error)
-			return []
-		})
-}
-
 export default {
 	name: "dataset-list",
+	components: {
+		PreservationState,
+		BusyButton,
+		DatasetVersionsModal,
+	},
 	data() {
 		return {
 			activeInModal: null,
 			fields: fields,
-			ownerSelect: null,
 			filterString: null,
-			myGroupsOptions: [],
-			showDatasetState: {
-				draft: true,
-				published: true,
-			},
+			showDatasetState: 'all',
 			isBusy: false,
 			error: null,
-			//highlighted: "0582f51d-86c3-2bc1-eb11-296b533b9731",
-			devWarning: process.env.VUE_APP_ENVIRONMENT === 'development' && !process.env.VUE_APP_QVAIN_API_URL,
+			devWarning: process.env.VUE_APP_ENVIRONMENT === 'development',
 			datasetList: [],
+			itemToBeDeleted: null
 		}
 	},
 	methods: {
-		// apiProvider fills the table with datasets from an (real or fake) API response.
-		// Function is passed (ctx, callback).
-		apiProvider(ctx) {
-			return apiProvider.bind(this)(ctx)
-		},
 		async fetchDataset() {
 			try {
 				this.error = null
-				const { data } = await (process.env.VUE_APP_ENVIRONMENT === 'development' && !process.env.VUE_APP_QVAIN_API_URL ?
+				const { data } = await (process.env.VUE_APP_ENVIRONMENT !== 'development' && !process.env.VUE_APP_QVAIN_API_URL ?
```

Comment:
I think this should be ===. 
I.e. fakeFetch if we are in development and don't have a backend, otherwise use the API.

Reply:
Yeah, you are correct. Do you think we could remove this face fetch feature not that we have the development environment running?

Reply:
Not a bad idea, let's remove it.